source/codegen/service_helpers.py

```diff
@@ -55,6 +55,33 @@ def create_args_for_callback(parameters):
     ])
 
 
+def create_standard_arg(parameter):
```

Comment:
Extracted this method so we could call it from the initial call in the `ivi_dance` case.

Reply:
`standard` as opposed to what? Is this just differentiating from the size mechanism args?

(Probably the best way to address this is a doc comment. But if a great name comes to you: even better)